### PR TITLE
gravatar improvements, componentification

### PR DESCRIPTION
**Pull request by nschloe**

### CODE REVIEW

**Comment by andrenarchy** [comment]
Veeery nice! :) Somehow the user's name is not displayed anymore in the navbar:

screenshot from 2016-02-08 13-12-10

Maybe that's an issue with closing tags?

**Comment by andrenarchy** [comment]
Also comments cannot be created anymore:

screenshot from 2016-02-08 13-14-17

**Comment by nschloe** [comment]
It works locally. Investigating...

**Comment by nschloe** [comment]
@andrenarchy All fixed now.

File: app/components/gravatar.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+export default function(app) {
+    app.component('gravatar', {
+      bindings: {
+        user: `<`,
+        size: '@',
+        classes: '@',
+      },
+      template: '<img width={{$ctrl.size}} height={{$ctrl.size}} ng-src="https://secure.gravatar.com/avatar/{{$ctrl.user.user.avatar.value}}?={{$ctrl.size}}&d=identicon" alt="{{$ctrl.user.displayName}} avatar" class="{{$ctrl.classes}}">',
```
Comment (andrenarchy) [inline app/components/gravatar.ts]: The `s` parameter is missing in the URL: `?s={{$ctrl.size}}...`

**Comment by andrenarchy** [comment]
Liest sich doch gut!

**Pull request merged**